pkg/manifests/csv.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: packageserver
   namespace: openshift-operator-lifecycle-manager
   labels:
-    olm.version: 0.19.0
+    olm.version: 0.0.0-24bb499733c06e67583c59cfe6f7a42ec0a996a9
```

Comment:
@perdasilva this probably happened because you ran `make manifests` which sets `OLM_VERSION` to commit based versioning.

You need to run `make verify-manifests` (or `make verify` for everything) and commit changes. `make verify-manifests` sets a static `0.19.0` version:

https://github.com/openshift/operator-framework-olm/blob/b85e89137df437381313f01762d6c8f870a7567e/Makefile#L153-L155

Reply:
Thank you!!!!